client/my-sites/guided-transfer/style.scss

```diff
@@ -0,0 +1,25 @@
+.guided-transfer {
```

Comment:
I use the nested style here because [this line](https://github.com/Automattic/wp-calypso/pull/6200/files#diff-aa32d032b7ed2abf670cc834214d8ad4R18) needs the extra css selector. Otherwise I'd have to use `!important`.